### ISSUE TITLE
Datetime error format when using DeleteApi

### ISSUE THREAD

**Issue by LaurentIsRunning** [influxdata/influxdb-client-csharp#167]
When calling the deleteapi to delete the content of a bucket, I get error regarding the datetime formatting:

Here is the call:
`FluxClient.GetDeleteApi().Delete(new DateTime(2018, 1, 1), new DateTime(2022,1,1), "", bucket, org).Wait();`

Here is the error message:
` One or more errors occurred. (invalid request; error parsing request json: invalid RFC3339Nano for field start, please format your time with RFC3339Nano format, example: 2009-01-02T23:00:00Z)`

 - Client Version: 1.16.0
 - InfluxDB Version: 2.0
 - Platform:

**Comment by bednar** [comment]
Hi @LaurentIsRunning,

thanks for using our client. The `DateTime` should be specified in `UTC`. Following code will be work:

```csharp
var start = new DateTime(2018, 1, 1).ToUniversalTime();
var stop = new DateTime(2022, 1, 1).ToUniversalTime();
FluxClient.GetDeleteApi().Delete(start, stop, "", bucket, org).Wait();
```

Regards

**Comment by LaurentIsRunning** [comment]
OK thanks!
Why do you let the user handle that? (you could check the Kind property of the datetime object for instance)
Do I have to convert all my datetime to UTC in general in the application?

**Comment by bednar** [comment]
> Why do you let the user handle that? 

It is not intention, but bug 😞 

> Do I have to convert all my datetime to UTC in general in the application?

The `Query` and `Write` properly convert the `DateTime` into UTC. We will prepare fix to correct handling DateTime in rest of our APIs.

**TD;LR**

https://github.com/influxdata/influxdb-client-csharp/blob/be8f5ac4188d0b0186a4366a77f701f2908bb92e/Client/InfluxDB.Client.Api/Client/ApiClient.cs#L32

should be updated to:

```csharp
private JsonSerializerSettings serializerSettings = new JsonSerializerSettings
{
    ConstructorHandling = ConstructorHandling.AllowNonPublicDefaultConstructor,
    DateTimeZoneHandling = DateTimeZoneHandling.Utc
};
```

**Comment by bednar** [comment]
@LaurentIsRunning, the issue is fixed in the Miletone [1.17.0](https://github.com/influxdata/influxdb-client-csharp/milestone/20), you could use preview version by:

```
dotnet add package InfluxDB.Client --version 1.17.0-dev.2402
```

https://www.nuget.org/packages/InfluxDB.Client/1.17.0-dev.2402
